service/options_test.go

```diff
@@ -43,7 +43,7 @@ func TestAddModules_Errors(t *testing.T) {
 }
 
 func TestWithLogger_OK(t *testing.T) {
-	logger := ulog.Logger()
+	logger := ulog.Builder().Build()
```

Comment:
Here is a revolutionary idea: how about `log.New()` which just does an empty builder?